File: Systems/lesson-1/add-numbers/add-numbers.c
#include <stdio.h>


int sum_array(int* numbers, int count) {
    int sum = 0;

    for (int i = 0; i < count; i++) {
        sum += numbers[i];
    }

    return sum;
}


int main() {
    int test_array[3] = { 0, 1 ,2 };
    int test_sum = sum_array(test_array, 3);

    printf("%d\n", test_sum);

    return 0;
}

File: Systems/lesson-1/add-numbers/add-numbers.js

function sum_array(numbers) {
    const count = numbers.length;
    let sum = 0;
    for (let i = 0; i < count; i++) {
        sum += numbers[i];
    }

    return sum;
}


function main() {
    let test_array = [0, 1, 2];
    const sum = sum_array(test_array);

    console.log(sum);
}


main();
